apps/mobile/src/features/ImportSigner/ImportSigner.container.tsx

```diff
@@ -27,20 +27,20 @@ export function ImportPrivateKey() {
         <View marginTop="$2">
           <SectionTitle
             paddingHorizontal={0}
-            title="Import a private key"
-            description="Enter your private key below. Make sure to do so in a safe and private place."
+            title="Import a signer"
+            description="Enter your private key or seed phrase below. Make sure to do so in a safe and private place."
           />
         </View>
 
         <YStack gap="$3" marginTop="$6" paddingVertical="$1">
           <View>
             <SafeInput
               height={114}
-              value={privateKey}
-              onChangeText={handlePrivateKeyChange}
+              value={input}
+              onChangeText={handleInputChange}
               placeholder="Paste here or type..."
               secureTextEntry={isMasked}
-              success={!!wallet}
+              success={!!wallet || inputType === 'seed-phrase'}
```

Comment:
shouldn't it be ` inputType === 'seed-phrase' && input.trim() !== ''` instead? Otherwise if the input is empty and the import type is seed-phrase we will always show success in this field, or am I missing something?